uber/site_sections/registration.py

```diff
@@ -55,13 +55,14 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
         attendees = attendees.order(order)
 
         groups = []
-        for group in session.query(Group) \
-                            .options(joinedload(Group.leader)) \
-                            .filter(Group.id.in_(
-                                session.query(Attendee.group_id)
-                                       .filter(Attendee.group_id != None, Attendee.first_name == '')
-                                       .distinct()
-                                       .subquery())).all():
+        for group in (session.query(Group)
+                             .options(joinedload(Group.leader))
+                             .filter(Group.status != c.WAITLISTED,
+                                     Group.id.in_(
+                                        session.query(Attendee.group_id)
+                                               .filter(Attendee.group_id != None, Attendee.first_name == '')
+                                               .distinct()
+                                               .subquery()))):
```

Comment:
So Danielle / Marketplace actuallu offers table for sale starting Sat. An for unclaimed tables.  Also, some of the Merchants will come ad attendees anyway, we wouldn't want to delete those badges.